doc/site/index.tex

```diff
@@ -464,37 +541,13 @@ \subsubsection{Windows using Strawberry Perl}\label{get.windows.strawberry}
 You may want to try \href{https://www.imagemagick.org/}{ImageMagick}, but
 you're on your own, there!  You may  have better luck with \texttt{Graphics::Magick}.
 
-\paragraph*{Installing \emph{only} prerequisites  under Strawberry}\label{get.windows.strawberry.prereq}\\
-Install \href{https://strawberryperl.com}{Strawberry Perl}, and
-the \TeX-system of your choice (if desired), 
-ImageMagick (if you want to try; see \ref{get.windows.imagemagick}),
-and then install the additional prerequisites as
+\paragraph*{Installi \emph{only} prerequisites  under Strawberry,}\label{get.windows.strawberry.prereq}
+except for \TeX, and ImageMagic (see \ref{get.windows.imagemagick}),
```

Comment:
a bunch of `ImageMagic` typos for `ImageMagick` with a trailing `k`, probably better to grep than for me to comment on each one.

Reply:
fixed.